modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/3_policies/page-definition.json

```
@@ -432,22 +434,14 @@
 																"pageElements": [
 																	{
 																		"definition": {
-																			"fragment": {
-																				"key": "fake-content"
-																			},
-																			"fragmentConfig": {
-																				"fakeContentId": "fake content",
-																				"height": "900px"
-																			},
-																			"fragmentFields": [
-																			],
-																			"fragmentStyle": {
-																				"marginTop": "4",
-																				"textAlign": ""
-																			},
-																			"indexed": true
+																			"widgetInstance": {
+																				"widgetConfig": {
+																					"properties": "route=policies-table"
+																				},
+																				"widgetName": "com_liferay_client_extension_web_internal_portlet_ClientExtensionEntryPortlet_20097_RAYLIFEAGENTPORTAL001"
```

Please, change this WidgetName value, because this information always change.

```suggestion
																				"widgetName": "[$CLIENT_EXTENSION_ENTRY_ID:RAYLIFEAGENTPORTAL$]"
```

---

Done!